assets/js/src/core/components/accordion/accordion.styles.ts

```diff
@@ -14,25 +14,126 @@
 import { createStyles } from 'antd-style'
 
 export const useStyles = createStyles(({ token, css }) => {
+  const themeToken = {
+    highlightBackgroundColor: '#F6FFED',
+    highlightBorderColor: '#B7EB8F',
+    highlightColor: '#52C41A',
+    signalBackgroundColor: '#E6F4FF',
+    signalBorderColor: '#91CAFF',
+    signalColor: '#1677FF',
+    ...token
+  }
+
   return {
```

Comment:
We should follow the BEM naming here. 

For example is the chance of having a success class in the code quite high.
With following BEM we can make sure that we don't get any unwanted side effects.

Reply:
Good point - I gave this a go int he last commit :) @vin0401